src/Runner.Worker/FileCommandManager.cs

```diff
@@ -322,21 +322,9 @@ public EnvFileKeyValuePairs(IExecutionContext context, string filePath)
                     var equalsIndex = line.IndexOf("=", StringComparison.Ordinal);
                     var heredocIndex = line.IndexOf("<<", StringComparison.Ordinal);
 
-                    // Normal style NAME=VALUE
-                    if (equalsIndex >= 0 && (heredocIndex < 0 || equalsIndex < heredocIndex))
```

Comment:
I basically just swapped the order of this conditional and the `else if` that follows it.